monkey/common/agent_event_serializers/register.py

```diff
@@ -9,3 +9,4 @@ def register_common_agent_event_serializers(
     event_serializer_registry[CredentialsStolenEvent] = PydanticAgentEventSerializer(
         CredentialsStolenEvent
     )
+    event_serializer_registry[PingScanEvent] = PydanticAgentEventSerializer(PingScanEvent)
```

Comment:
There should be a `PydanticAgentEventSerializerRegistry` which would serialize all events with `PydanticAgentEventSerializer` by default. There's no point in having to manually register each event

Reply:
Sounds good. Create an issue.

Reply:
https://github.com/guardicore/monkey/issues/2378